src/components/Card.js

```diff
@@ -47,7 +73,9 @@ export default class Card {
     this._cardImage.alt = this._name;
 
     this._setEventsListeners();
-
+    if (this._isLiked) {
```

Comment:
Needs correction: Please use `this. _renderLikes()` instead of lines 76-78, since there should not be code duplications